backend/akr/src/main/java/fi/oph/akr/service/ContactRequestService.java

```diff
@@ -113,29 +124,29 @@ private void saveContactRequestTranslators(final List<Translator> translators, f
 
   private void saveContactRequestEmails(
     final ContactRequestDTO contactRequestDTO,
-    final List<PersonalData> translatorsPersonalDatas,
+    final List<TranslatorPersonalDataDTO> translatorsPersonalDataDTOs,
     final List<Translator> translators
   ) {
-    final List<PersonalData> translatorsPersonalDatasWithEmail = translatorsPersonalDatas
+    final List<TranslatorPersonalDataDTO> translatorsPersonalDatasWithEmail = translatorsPersonalDataDTOs
       .stream()
       .filter(t -> t.getEmail() != null)
       .collect(Collectors.toCollection(ArrayList::new));
 
-    final List<PersonalData> translatorsPersonalDatasWithoutEmail = translatorsPersonalDatas
+    final List<TranslatorPersonalDataDTO> translatorsPersonalDatasWithoutEmail = translatorsPersonalDataDTOs
       .stream()
       .filter(t -> t.getEmail() == null)
       .collect(Collectors.toCollection(ArrayList::new));
 
     sendTranslatorEmails(translatorsPersonalDatasWithEmail, contactRequestDTO);
-    sendRequesterEmail(translatorsPersonalDatas, contactRequestDTO);
+    sendRequesterEmail(translatorsPersonalDataDTOs, contactRequestDTO);
 
     if (!translatorsPersonalDatasWithoutEmail.isEmpty()) {
-      sendClerkEmail(translators, contactRequestDTO);
+      sendClerkEmail(translatorsPersonalDatasWithoutEmail, contactRequestDTO);
```

Comment:
Tämä muutos lienee se oleellinen osuus. Kaikki muu taitaa olla vain seremoniaa uuden DTO:n lisäykseen liittyen?

Reply:
Juuri näin. Muu on refaktorointia, koska sendClerkEmail-metodin sisällä tarvitaan sekä tietoa Translator:sta että PersonalData:sta, joten ne on yhdistetty omaan DTO:hon.